rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -79,7 +79,8 @@ class QuotingPanel {
   }
 
   addListeners() {
-    document.getElementById('SubmitQuote').addEventListener('submit', (event) => { event.preventDefault() });
+    document.getElementById("SubmitQuote").addEventListener('submit', (event) => { event.preventDefault() });
+    document.getElementById("searchAddressButton").addEventListener('click', this.handleOpenQuoteSubmission);
```

Comment:
Same here, the handler name should not be about what happens _within_ the handler, but rather about what it's handling, in this case it's handling the search Address button click, so maybe `handleSearchAddressClick`

Reply:
done